src/python/m5/params.py

```diff
@@ -882,9 +882,6 @@ def __str__(self):
     def __eq__(self, other):
         return self.pc == other.get_pc() and self.count == other.get_count()
 
-    def __hash__(self):
-        return hash((int(self.pc), int(self.count)))
-
```

Comment:
Why this change?

Reply:
Arg... this was the cause of a bug I've been trying to figure out for 20 minutes. Yeah, I didn't mean to remove this :).